### PR TITLE
Included new function for Handlebars Helper + Updated Doco

### PR DESCRIPTION
**Pull request by kachihro**
Text added to README - with example

### CODE REVIEW

File: docs/extensibility/templating.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The following custom helpers are available in addition to the [handlebars-helper
...
 | `{{getDate <data_value> "<format>" <time handling>}}` | Format the date with [Moment.js](https://momentjs.com/docs/#/parsing/string-format/) according to the current language. Date in the managed property should be on the form `2018-09-10T06:29:25.0000000Z` for the function to work.<p>&lt;time handling&gt; is optional and takes <ul><li>0 = format to browsers time zone (default)</li><li>1 = ignore Z time and handle as browsers local time zone</li><li>2 = strip time and set to 00:00:00 in browsers local time zone</li><li>3 = display in the time zone for the current web</li><li>4 = display in the time zone from the uers profile</li> | `{{getDate 2018-09-10T06:29:25.0000000Z 'LL'}}`
-| `{{getUrlField <managed_propertyOWSURLH> "URL|Title"}}` | _Use with SharePoint Search data source_. Returns the URL or Title part of a URL autocreated managed property | `{{getUrlField MyPropertyOWSURLH "Title"}}`
+| `{{getUrlField <managed_propertyOWSURLH> "URL/Title"}}` | _Use with SharePoint Search data source_. Returns the URL or Title part of a URL autocreated managed property | `{{getUrlField MyPropertyOWSURLH "Title"}}`
```
Comment (wobba) [inline docs/extensibility/templating.md:44]: I'd rather leave this a pipe and not a slash.

File: docs/extensibility/templating.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ The following custom helpers are available in addition to the [handlebars-helper
 | `{{regex <expression> <string>}}` | Run a regular expression and return the match. Return `-` if not match. | `{{regex '\d+' 'digit 15 is a number'}}` will return `15`.
 | `{{getPageContext "<SPFx page property>"}}` | Retrieve a property from the SPFx context object. | `{{getPageContext "user.displayName"}}` or `{{getPageContext "cultureInfo.currentUICultureName"}}`
 | `{{getAttachments}}`| Return object structure for list item attachments. | `{{#getAttachments LinkOfficeChild}}<a href="{{url}}">{{index}} - {{fileName}}</href>{{/getAttachments}}`
+| `{{dayDiff "date1 YYYY-MM-DD" "date2 YYYY-MM-DD"}}`| Return the number of days between two dates, eg. show an icon for files created within the last 30 days. | `{{#compare (dayDiff (getDate Created 'YYYY-MMM-DD' ) (getDate timestamp 'YYYY-MMM-DD' )) "<=" 30 }} <img src='/SiteAssets/New.png' />{{/compare}}`
```
Comment (wobba) [inline docs/extensibility/templating.md:54]: Seems your use of quotes in the syntax does not match your example. Should it be `dayDiff <date1> <date2>`?

File: docs/extensibility/templating.md (hunk 1 above)
Comment (kachihro) [reply docs/extensibility/templating.md:44]: Syntax is correct to use PIPE - but it breaks the table in MarkDown (MD)

File: docs/extensibility/templating.md (hunk 2 above)
Comment (kachihro) [reply docs/extensibility/templating.md]: Have updated this - thanks

**Comment by wobba** [comment]
Thank you :)

**Pull request merged**